modules/aws/vpc/vpc.tf

```diff
@@ -114,7 +114,8 @@ resource "aws_route_table" "cluster_vpc_private" {
   tags = merge(
     local.common_tags,
     map(
-      "Name", "${var.cluster_name}_private_${count.index}"
+      "Name", "${var.cluster_name}_private_${count.index}",
+      "giantswarm.io/route-table-type", "private"
```

Comment:
How about `giantswarm.io/cluster`?

Reply:
It's already there in `local.common_tags` https://github.com/giantswarm/giantnetes-terraform/blob/master/modules/aws/vpc/vpc.tf#L11